<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentMainWindow</class>
 <widget class="QWidget" name="StudentMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student Dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="leftLayout">
      <item>
       <widget class="QPushButton" name="registerTermButton">
        <property name="text">
         <string>ثبت نام ترم</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="viewInfoButton">
        <property name="text">
         <string>مشاهده اطلاعات</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeCourseButton">
        <property name="text">
         <string>حذف درس</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="viewFinalExamButton">
        <property name="text">
         <string>مشاهده زمان پایان‌ترم</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>refresh</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="courseListWidget">
      <property name="toolTip">
       <string>لیست درس‌های دانشجو</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
